MantidQt/CustomInterfaces/inc/MantidQtCustomInterfaces/Indirect/IndirectSqw.ui

```diff
@@ -138,6 +138,19 @@
           </property>
          </widget>
         </item>
+        <item row="0" column="0" colspan="2">
+         <widget class="QLabel" name="lbMethod">
+          <property name="minimumSize">
+           <size>
+            <width>43</width>
+            <height>0</height>
+           </size>
+          </property>
+          <property name="text">
+           <string>The method used for Q Rebinning is Normalised Polygon</string>
```

Comment:
I'm not sure this offers much to the average user. We don't have explanations of methods for other parts of the interface so I'd say we just remove this label entirely and just leave the bin inputs.